nomad-gcp/variables.tf

```diff
@@ -186,7 +186,7 @@ variable "machine_image_project" {
 variable "machine_image_family" {
   type        = string
   description = "The family value used to retrieve the virtual machine image."
-  default     = "ubuntu-2004-lts"
+  default     = "ubuntu-2024-lts"
```

Comment:
Should this be `ubuntu-2204-lst-amd64`? The image family here doesn't appear to exist.

Reply:
:point_up: And when I use this image family, jobs appear to fail because Docker seems to not be installed, although the startup script seems to run fine...

![Screenshot from 2025-05-09 08-55-27](https://github.com/user-attachments/assets/c2f025c9-70a7-4b22-b9b8-188502deb995)

Reply:
Yes, that should be ubuntu-2404 not ubuntu 2024.